tests/e2e/e2e_test.go

```diff
@@ -211,3 +212,33 @@ func (s *IntegrationTestSuite) TestStateSync() {
 	err = chain.RemoveNode(stateSynchingNode.Name)
 	s.NoError(err)
 }
+
+func (s *IntegrationTestSuite) TestExpeditedProposals() {
+	if !s.skipUpgrade {
+		s.T().Skip("this can be re-enabled post v12")
+	}
+
+	var wg sync.WaitGroup
+	chain := s.configurer.GetChainConfig(0)
+	node, err := chain.GetDefaultNode()
+	s.NoError(err)
+
+	node.SubmitTextProposal("expedited text proposal", sdk.NewCoin(appparams.BaseCoinUnit, sdk.NewInt(config.InitialMinExpeditedDeposit)), true)
+	chain.LatestProposalNumber += 1
+	node.DepositProposal(chain.LatestProposalNumber, true)
+	wg.Add(1)
+	totalTime := make(chan time.Duration, 1)
+	go node.QueryPropStatusTimed(&wg, chain.LatestProposalNumber, "PROPOSAL_STATUS_PASSED", totalTime)
+	for _, node := range chain.NodeConfigs {
+		node.VoteYesProposal(initialization.ValidatorWalletName, chain.LatestProposalNumber)
+	}
+	// wait till prop status reaches desired pass status
+	wg.Wait()
+	elapsed := <-totalTime
+	// compare the time it took to reach pass status to expected expedited voting period
+	expeditedVotingPeriodDuration := time.Duration(chain.ExpeditedVotingPeriod * 1000000000)
+	timeDelta := elapsed - expeditedVotingPeriodDuration
+	// ensure delta is within one second of expected time
+	s.Require().Less(timeDelta, time.Second)
```

Comment:
Can we have `time.Second` as a constant that we also use to set the parameter on chain? Just to make the link clear between how we configure it and the assertion here

Reply:
I don't think we would want this to be a set parameter right? I am basically just saying that the expeditedVotingPeriodDuration that this go routine calculates should realistically be within a second of the actual on chain param. I might be misunderstanding your recommendation here though

Reply:
Sorry my comment ended up being confusing.

What I meant was let's have a global constant defined for `expeditedVotingPeriodDuration`, `votingPeriodDuration`.

And also reuse those constants here:
https://github.com/osmosis-labs/osmosis/blob/6c65ea106b402245788f1390dbdfdbc9229fdfa6/tests/e2e/configurer/current.go#L48

Basically, the goal is to avoid hardcoding these values in different places. If we decide to change them in the future, we might update in one place but forget about another place and start having weird issues.

Having shared constants defined for such things helps mitigate these problems

Reply:
Actually, nvm I think that's already the case since we are using `chainConfig.ExpeditedVotingPeriod`

Reply:
The `*1000000000` part misled me. I'm going to resolve this, sorry for the confusion